wasm-node/src/lib.rs

```diff
@@ -2,3 +2,4 @@
 
 pub mod node;
 pub mod utils;
+pub mod wrapper;
```

Comment:
I don't think user needs to know that it is wrapped or from which library is wrapped. We can reexport the types in `node` module

Reply:
true, removed pub here. 

wasm-bindgen does seem to care only about type visibility and doesn't evaluate whether it's entire path is reachable, so we don't need to re-export it for js sake, and I hope we won't use them in other rust crates :p 
Also note that in exported wasm module everything is dumped in top scope, so our internal layout isn't reflected.